ingest-app/docs/api.md

```diff
@@ -288,15 +288,23 @@ A native-id is an identifier, unique per provider, used to identify concepts wit
 
 The following are the latest acceptable UMM schema versions for metadata ingest:
 
-<p>UMM-C: {{ umm-c }}<br/>UMM-G: {{ umm-g }}<br/>UMM-S: {{ umm-s }}<br/>UMM-T: {{ umm-t }}<br/>UMM-SUB: {{ umm-sub }}<br/>UMM-VAR: {{ umm-var }}</p>
+* UMM-C: {{ umm-c }}
+* UMM-G: {{ umm-g }}
+* UMM-S: {{ umm-s }}
+* UMM-T: {{ umm-t }}
+* UMM-SUB: {{ umm-sub }}
+* UMM-VAR: {{ umm-var }}
```

Comment:
Why are you changing this?  That's the look we want in the html.

Reply:
I feel strongly that HTML should not be in markdown. We should live with HTML unless there is a compelling reason to use HTML. There is not in this case.